src/Bicep.Cli.IntegrationTests/BuildParamsCommandTests.cs

```diff
@@ -363,7 +363,28 @@ public async Task Build_params_to_stdout_with_non_bicep_references_should_succee
             var settings = new InvocationSettings(new(TestContext, RegistryEnabled: true), clients.ContainerRegistry, clients.TemplateSpec);
 
             var result = await Bicep(settings, "build-params", baselineFolder.EntryFile.OutputFilePath, "--stdout");
-            result.Should().Succeed().And.NotHaveStderr();
+            result.Should().Succeed();
+
+            var parametersStdout = result.Stdout.FromJson<BuildParamsStdout>();
+            // Force consistency for escaped newlines.
+            parametersStdout = parametersStdout with { templateJson = parametersStdout?.templateJson?.ReplaceLineEndings("\n") };
+            outputFile.WriteJsonToOutputFolder(parametersStdout);
+            outputFile.ShouldHaveExpectedJsonValue();
+        }
+
+        [TestMethod]
+        [EmbeddedFilesTestData(@"Files/BuildParamsCommandTests/Extends/main\.bicepparam")]
```

Comment:
Won't this be already covered by the previous test? `Build_params_to_stdout_with_non_bicep_references_should_succeed`

Reply:
I changed the file we're testing and make it work 👍 thanks.